.gitlab-ci.yml

```diff
@@ -76,19 +76,29 @@ nightly-test:
   <<:                              *docker-env
   stage:                           test
   variables:
-    EXTRA_FEATURES:                "unstable"
+    FEATURES:                      "unstable"
   script:
     - cargo +nightly build --verbose --features="${EXTRA_FEATURES}" || exit 1
-    - RUST_BACKTRACE=1 cargo +nightly test --workspace --verbose --features="${EXTRA_FEATURES}"
+    - cargo +nightly test --workspace --verbose --features="${EXTRA_FEATURES}"
```

Comment:
this doesn't use `FEATURES` var from above, is it intended?

Reply:
oh woops, it definitely isn't intended, thanks for pointing that out! Let me open a PR real quick